src/aktualizr_primary/main.cc

```diff
@@ -147,10 +147,10 @@ int main(int argc, char *argv[]) {
     } else {
       aktualizr.RunForever().get();
     }
-    r = 0;
+    r = EXIT_SUCCESS;
   } catch (const std::exception &ex) {
     LOG_ERROR << ex.what();
-    r = -1;
+    r = EXIT_FAILURE;
```

Comment:
It's already `= EXIT_FAILURE`, right?

Reply:
Yeah, this line can be removed.

Reply:
Do we need the `int r` at all? There are only two exit points, so maybe just `return EXIT_FAILURE`?